api/database/services.py

```diff
@@ -0,0 +1,144 @@
+import os
+from flask import Flask, request, jsonify
+from flask_pymongo import PyMongo
+import json
+
+def index():
+    """Display beacon info."""
+    beacon_info = {
+        "id": ".".join(reversed(request.host.split("."))),
+        "name": "Imaging beacon",
+        "type": {'group': f'{"test"}',
+                    'artifact': f'{"beacon"}',
+                    'version': f'{"0.0.0"}'},
+        "description": "bp test beacon",
+        "organization": {
+            "name": "",
+            "url": "",
+        },
+        "contactUrl": "",
+        "documentationUrl": "",
+        "createdAt": "",
+        "updatedAt": "",
+        "environment": "",
+        "version": "",
+    }
+    return beacon_info
+
+def getItems(db):
+    """List all db items."""
+    dbImages = db.images.find()
+    dbDataset = db.dataset.find()
+    dbSamples = list(db.sample.find())
+    for coll in db.list_collection_names():
+        print(coll)
+    datasets = []
+    for set in dbDataset:
+        setdata = {"id": str(set["_id"]), "alias": set["alias"], "attributes": set["attributes"], "title": set["title"], "description": set["description"], "datasetType": set["datasetType"], "policyRef": set["policyRef"], "imageRef": ["imageRef"]}
+        datasets.append(setdata)
```

Comment:
```suggestion
    for ds in dbDataset:
        setdata = {"id": str(ds["_id"]), "alias": ds["alias"], "attributes": ds["attributes"], "title": ds["title"], "description": ds["description"], "datasetType": ds["datasetType"], "policyRef": ds["policyRef"], "imageRef": ["imageRef"]}
        datasets.append(setdata)
```
avoid using reserved keywords like `set`